lib/hatchet/heroku_run.rb

```diff
@@ -65,6 +67,7 @@ def call
           message << "\nreleases: #{@app.releases}"
           message << "\n#{caller.join("\n")}"
           @stderr.puts message
+          sleep(@retry_delay) # without run_multi, this will prevent occasional "can only run one free dyno" errors
```

Comment:
*Without*. I ran into this myself when testing. An immediate retry on a non-`run_multi`'d app (which is not always desirable because it requires a `web` process), I'd get these errors, because API thinks the previous `heroku run` dyno is still up.